schutzbot/build.sh

```diff
@@ -19,8 +19,17 @@ if [[ $ID == rhel ]] && ! rpm -q epel-release; then
 fi
 
 # Register RHEL if we are provided with a registration script.
-if [[ -n "${RHN_REGISTRATION_SCRIPT:-}" ]] && ! sudo subscription-manager status; then
-    greenprint "🪙 Registering RHEL instance"
+# if [[ -n "${RHN_REGISTRATION_SCRIPT:-}" ]] && ! sudo subscription-manager status; then
+#     greenprint "🪙 Registering RHEL instance"
+#     sudo chmod +x "$RHN_REGISTRATION_SCRIPT"
+#     sudo "$RHN_REGISTRATION_SCRIPT"
+# fi
```

Comment:
This looks like a left over given the changes a few lines below.

Reply:
Hmm... don't know why it looks like this. Will try to fix it.

Reply:
Fixed, thanks.